plugins/destination/managed_writer.go

```diff
@@ -32,31 +32,27 @@ func (p *Plugin) worker(ctx context.Context, metrics *Metrics, table *schema.Tab
 				}
 				return
 			}
-			if len(resources) == p.spec.BatchSize || sizeBytes+r.Size() > p.spec.BatchSizeBytes {
+			if len(resources) == p.spec.BatchSize {
```

Comment:
We seem to be missing the condition for flushing on byte size here, compared to what it was before. Is this intentional?

Reply:
There is no `record.Len` method but I'll add it right now (it exist in the python implementation and should be relatively straight forward)